packages/typespec-ts/test/modularUnit/scenarios/samples/operations/multipleExamplesInOneFile.md

```diff
@@ -189,19 +185,15 @@ async function employeesCreateOrUpdateMinage(): Promise<void> {
   const credential = new DefaultAzureCredential();
   const subscriptionId = "11809CA1-E126-4017-945E-AA795CD5C5A9";
   const client = new ContosoClient(credential, subscriptionId);
-  const result = await client.employees.createOrUpdate(
-    "rgopenapi",
-    "9KF-f-8b",
-    {
-      properties: {
-        age: 1,
-        city: "gydhnntudughbmxlkyzrskcdkotrxn",
-        profile: "ms",
-      },
-      tags: { key2913: "urperxmkkhhkp" },
-      location: "itajgxyqozseoygnl",
+  const result = await client.createOrUpdate("rgopenapi", "9KF-f-8b", {
```

Comment:
@MaryGao Could you help check if we need to manual set hierarchyClient true or enableOperationGroup true for those unit test?

Reply:
yes, this is not only for sample but also for all UTs because we didn't adopt the transformRLCOption which have any default setting logic.

not the current priority but i think we could refactor this in future.